package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "infusion-nexus",
-    "version": "0.4.0",
+    "version": "0.5.0",
```

Comment:
I think as a breaking API change this warrants an major version bump - we may as well declare that this thing is a thing now